build.gradle

```diff
@@ -9,8 +9,12 @@ plugins {
 }
 
 repositories {
-    maven { url 'https://repository.jboss.org' }
+    mavenLocal()
     mavenCentral()
+    maven { url 'https://repository.jboss.org' }
+    maven {
+        url 'https://packages.jetbrains.team/maven/p/ij/intellij-dependencies'
```

Comment:
Why is this required ?

Reply:
Part of setup guide: https://github.com/redhat-developer/intellij-common-ui-test-library#step-1-adding-repositories